en/services/opendroneid.md

```diff
@@ -5,24 +5,11 @@
 
 ## Relevant specifications {#specifications}
 
-### United States {#us}
+The MAVLink messages defined for usage with Open Drone ID are compliant with the following standards:
+* ASTM F3411 Specification for Remote ID and Tracking
```

Comment:
@friissoren Happy with all these changes. Is there a link to these we can use here? Save people time googling.

Reply:
I'll merge now on the assumption that the answer is "no" but would appreciate follow up PR if you think it is reasonable.

Reply:
The answer is yes. The link was hidden in the same PR :-)
https://github.com/mavlink/mavlink-devguide/blob/master/en/services/opendroneid.md?plain=1#L12

I decided it is easier to maintain that information in one place instead of two. I have some changes to the info behind the link pending, that I will be merging in soon.